webpack.config.js

```diff
@@ -205,27 +206,12 @@ function getWebpackConfig( {
 			// https://github.com/localForage/localForage/issues/577
 			noParse: /[\/\\]node_modules[\/\\]localforage[\/\\]dist[\/\\]localforage\.js$/,
 			rules: [
-				{
-					test: /\.jsx?$/,
-					exclude: /node_modules\//,
-					use: [
-						{
-							loader: 'thread-loader',
-							options: {
-								workers: workerCount,
-							},
-						},
-						{
-							loader: 'babel-loader',
-							options: {
-								configFile: path.resolve( __dirname, 'babel.config.js' ),
-								babelrc: false,
-								cacheDirectory: path.join( __dirname, 'build', '.babel-client-cache' ),
-								cacheIdentifier,
-							},
-						},
-					],
-				},
+				JsxConfig.loader( {
```

Comment:
I'm opposed to calling this a `JSX` loader. JSX generally refers to HTMLish-in-JS syntax popularized by React. Especially within Babel, that refers to specific syntax and transforms.

It _may_ handle JSX, but that depends on the babel config it receives.

I'd prefer something related to JavaScript, or esnext or something that describes turning non-standard JS into something browser compatible.

Reply:
Or `transpile` or `babel` or...?

Reply:
`transpileJs` or `compileJs` seem reasonable

Reply:
:+1:

Reply:
`transpileJs` works for me.